examples/add_vectors_image.py

```diff
@@ -15,6 +15,10 @@
     # create the viewer and window
     viewer = ViewerApp()
 
+    image = 0.2*np.random.rand(24, 100) + 0.5
```

Comment:
Generally I prefer `np.random.random((24, 100))` as it is consistent with other NumPy functions (zeros, ones, full, etc).

Reply:
fixed

Reply:
Nope! I get the following error
```python
    image = 0.2*np.random.rand((24, 100)) + 0.5
  File "mtrand.pyx", line 1365, in mtrand.RandomState.rand
  File "mtrand.pyx", line 861, in mtrand.RandomState.random_sample
  File "mtrand.pyx", line 167, in mtrand.cont0_array
TypeError: 'tuple' object cannot be interpreted as an integer
```

Reply:
ah - i see the difference now - fixed :-)

Reply:
=)